test/phone.js

```diff
@@ -0,0 +1,53 @@
+require('dotenv').config({
```

Comment:
As a general comment, let's use 2 spaces per indent.  We haven't set up linting yet but it's likely we'll use the [airbnb style guide](https://github.com/airbnb/javascript)